src/pykiso/interfaces/thread_auxiliary.py

```diff
@@ -205,9 +207,6 @@ def run(self) -> None:
             if not self.is_instance:
                 time.sleep(0.050)
 
-            # If auxiliary instance is created and is pausable
```

Comment:
Not related to this, but just a few more findings regarding the cleanup (sorry I forgot to mention these last time):
- in auxiliary.py: docstrings of abcs create_instance and delete_instance are wrong, they should be something like `"""Handle auxiliary creation/deletion."""`
- in auxiliary.py: error log of suspend/resume can now be more specific, like `log.error(f"Auxiliary '{self}' is already running/stopped")`
- in thread_auxiliary and mp_auxiliary: attribute `self.wait_event` can be removed